### ISSUE TITLE
Call to undefined function GuzzleHttp\Psr7\build_query() 

### ISSUE THREAD

**Issue by Vitexus** [FitecoGroup/php-kimai-api-client#2]
The new Guzzle does not use build_query anymore

```
PHP Fatal error:  Uncaught Error: Call to undefined function GuzzleHttp\Psr7\build_query() in /home/vitex/Projects/Sizek/kimai2abraflexi/vendor/fiteco-group/kimai-client/src/Api/ActivityApi.php:472
Stack trace:
#0 /home/vitex/Projects/Sizek/kimai2abraflexi/vendor/fiteco-group/kimai-client/src/Api/ActivityApi.php(166): Fiteco\KimaiClient\Api\ActivityApi->apiActivitiesGetRequest()
#1 /home/vitex/Projects/Sizek/kimai2abraflexi/vendor/fiteco-group/kimai-client/src/Api/ActivityApi.php(140): Fiteco\KimaiClient\Api\ActivityApi->apiActivitiesGetWithHttpInfo()
#2 /home/vitex/Projects/Sizek/kimai2abraflexi/src/Kimai2AbraFlexi/Importer.php(81): Fiteco\KimaiClient\Api\ActivityApi->apiActivitiesGet()
#3 /home/vitex/Projects/Sizek/kimai2abraflexi/src/kiami2abraflexi.php(22): Kimai2AbraFlexi\Importer->__construct()
#4 {main}
  thrown in /home/vitex/Projects/Sizek/kimai2abraflexi/vendor/fiteco-group/kimai-client/src/Api/ActivityApi.php on line 472
```

**Comment by NikoGrano** [comment]
Thanks you for the PR and Issue. However I need to confirm few things first.

1. What version of `guzzlehttp/guzzle` are you using in the project?
2. Does PR support generation of sources with openapi-generator.
3. If not, there should be PR made for openapi-generator also to fix this.

**Comment by NikoGrano** [comment]
Just to answer my own question:

Using `guzzlehttp/guzzle@6.5.x`: Requires `guzzlehttp/psr7@1.6.1`, which doesn't have `\GuzzleHttp\Psr7\Query` implemented. The implementation of `\GuzzleHttp\Psr7\Query` is available at earliest on `1.7.0`. (Also 7.0 is having this issue)

As I'm not fan of keeping stuff legacy depedent I'll make following changes:
- Add `"guzzlehttp/psr7": "^1.7 || ^2.0",` to `composer.json`. 

This is what `guzzlehttp/guzzle` is at this moment [also doing](https://github.com/guzzle/guzzle/blob/1482295aa64d042f9f5b9a557cd1e0518d419e12/composer.json#L34) in their `master` branch.

**Comment by NikoGrano** [comment]
Release 0.2.0 is now available via composer.
